share/man/man7/development.7

```diff
@@ -116,7 +119,14 @@ build and test status of CURRENT and STABLE branches,
 the continuous integration system is at:
 .Pp
 .Lk https://ci.FreeBSD.org
-.Pp
+.Sh FILES
+.Bl -tag -width "src/tools/tools/git/HOWTO" -compact
+.It Pa src/CONTRIBUTING.md
+.Fx
+contribution guidelines
+.It Pa src/tools/tools/git/HOWTO
```

Comment:
IMO we should not link to this, it is stale. If desired we could extract useful content into somewhere less hidden.

Reply:
Rationale: doc and ports trees don't accept gihub PRs. I had patches sitting in there for months, but then when I discovered how to put them on phabricator, the discussions began immediately. Then from there, git-arc(1) took me even longer to discover and is actually cozy.

I'm open to any suggestions how to solve this problem in a better way, but that was the problem I'm trying to solve.

Reply:
Right, but the `arcgit` tool described in the `HOWTO` is something older and separate from `git-arc`, but with similar functionality. It is a step backwards to start advertising it now.

Reply:
Yea. I'd delete that file even i think and put everything in contributing.md

Reply:
Thanks so much for your time!

Reply:
Id keep contributing. Sorry if that wasn't clear. Just the howto file that's a problem

Reply:
I'd like to mention git-arc. Is it more appropriate:

1. Files: git-arc.sh
2. Files: git-arc.1
3. See also: /path/git-arc.1